### PR TITLE
USWDS-Site: Process List Accessibility Checklist

### LINKED ISSUE #2902: USWDS-Site: Publish process list accessibility checklists page
## Summary
Publish the `usa-process-list` accessibility test checklist items found in the [Critical Checklist spreadsheet](https://docs.google.com/spreadsheets/d/1fAJaObQHv7aIaeUpH4fTLmauhmugMtNCrbpZe4BJcPY/edit?usp=sharing) (Google Doc :lock:). 

The process for creating a new accessibility test page can be found in the [Creating accessibility checklists - process doc](https://docs.google.com/document/d/14WTCsdzgwpoz_Miztd66RmEGfwwSEUFCVN1DkSQITTE/edit#heading=h.fa3sv1ray42g) (Google Doc :lock:).

### PR DESCRIPTION
**Pull request by RachelCorsino**
# Summary

Added accessibility test page for process list component

> [!Important]
> We need to update changelog dates before merge

## Related issue

Closes #2902 

## Preview link
[Process List Component Page](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/rc-process-list-checklist/components/process-list/)
[Process List Accessibility Tests Page](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/rc-process-list-checklist/components/process-list/accessibility-tests/)

## Testing and review
Follow these steps:

1. Confirm that both the main component page and the accessibility tests page have the correct compliance tag at the top.
2. Confirm that the main component page links to the accessibility tests page in the side navigation.
3. Confirm that the main component page links to the accessibility tests page from the accessibility guidance section.
4. Confirm that the main component page shows the accessibility tests summary.
5. Check that accessibility tests page provides the correct counts for each test status type.
6. Confirm the test checklist summaries and data are accurate.
7. Confirm no visual issues.
8. Confirm there is an appropriate changelog entry on both the main component page and the accessibility tests page.

### CODE REVIEW

File: _data/accessibility-tests/process-list.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+title: Process list
+component_status: pass
+test_items:
+# General tests
+  - summary: Headings are structured in a logical order.
+    summary_additional: When you interact with a process list, you understand headings move in a logical order. For example, headings move from h2 to h3.
+    test_status: conditional
+    test_type: general
+    version_tested: 1.3.1
+    wcag_criterion: 3.10.0
```
Comment (amyleadem) [inline _data/accessibility-tests/process-list.yml:10]: **issue:** Lil data mixup
```suggestion
    version_tested: 3.10.0
    wcag_criterion: 1.3.1
```

File: _data/accessibility-tests/process-list.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
+title: Process list
+component_status: pass
+test_items:
+# General tests
+  - summary: Headings are structured in a logical order.
+    summary_additional: When you interact with a process list, you understand headings move in a logical order. For example, headings move from h2 to h3.
```
Comment (amyleadem) [inline _data/accessibility-tests/process-list.yml:6]: **suggestion:** Wrap code references in backticks

```suggestion
    summary_additional: When you interact with a process list, you understand headings move in a logical order. For example, headings move from `h2` to `h3`.
```

File: _data/accessibility-tests/process-list.yml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    test_status: pass
+    test_type: screen_reader
+    version_tested: 3.8.2
+    wcag_criterion: 1.3.1
+  - summary: Screen reader announces location of process list and order of steps to match visual order on screen.
+    summary_additional: When you use a screen reader to tab into the process list, the number of list items are announced before the first step in the process is described.
+    test_status: conditional
+    test_type: screen_reader
+    version_tested: 3.8.2
+    wcag_criterion: 1.3.2
```
Comment (amyleadem) [inline _data/accessibility-tests/process-list.yml:41]: **polish:** Format code
```suggestion
    wcag_criterion: 1.3.2
    
```

File: _data/changelogs/component-process-list-accessibility.yml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+title: Process list accessibility tests
+type: component
+items:
+ - date: NNNN-NN-NN
+   summary: Added accessibility tests page.
+   affectsGuidance: true
+   githubPr: 3007
+   githubRepo: uswds-site
```
Comment (amyleadem) [inline _data/changelogs/component-process-list-accessibility.yml:8]: **polish:** Format code

```suggestion
   githubRepo: uswds-site
   
```

**Review by amyleadem — CHANGES_REQUESTED** [review]
Thanks for the speedy work on this @RachelCorsino!
Added a few very small suggestions below. Other than that, it's looking good.

File: _data/changelogs/component-process-list-accessibility.yml (hunk 4 above)
Comment (RachelCorsino) [reply _data/changelogs/component-process-list-accessibility.yml]: Resolved in [73ddb50](https://github.com/uswds/uswds-site/pull/3007/commits/73ddb50d7c1d23da0cfc280840136c43b0245c75)

File: _data/accessibility-tests/process-list.yml (hunk 2 above)
Comment (RachelCorsino) [reply _data/accessibility-tests/process-list.yml]: Resolved in [73ddb50](https://github.com/uswds/uswds-site/pull/3007/commits/73ddb50d7c1d23da0cfc280840136c43b0245c75)

File: _data/accessibility-tests/process-list.yml (hunk 1 above)
Comment (RachelCorsino) [reply _data/accessibility-tests/process-list.yml]: Resolved in [73ddb50](https://github.com/uswds/uswds-site/pull/3007/commits/73ddb50d7c1d23da0cfc280840136c43b0245c75)

**Review by mahoneycm — CHANGES_REQUESTED** [review]
Looks good @RachelCorsino ! Just echoing Amy's suggestion to reorder the items by criterion number within each section. Thanks!

**Comment by RachelCorsino** [comment]
> Looks good @RachelCorsino ! Just echoing Amy's suggestion to reorder the items by criterion number within each section. Thanks!


> @RachelCorsino, I've gone through everything and looks good. I won't reiterate the change @amyleadem and @mahoneycm already mentioned. :)

Resolved in [625f8d9](https://github.com/uswds/uswds-site/pull/3007/commits/625f8d9265a3a3254b527d79ccf40720a599d49d). Tests are in the correct order now!

**Review by mahoneycm — DISMISSED** [review]
Thanks Rachel!

**Review by amyleadem — DISMISSED** [review]
LGTM!

**Review by annepetersen — APPROVED** [review]
LGTM

**Pull request merged**